CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.13)
```

Comment:
Can you change the minimum version to 2.8.12? That's the version Trusty ships with.